pkg/daemon/ceph/osd/daemon.go

```diff
@@ -55,10 +55,11 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID string, pvcBack
 		}
 
 		context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-an")
-
-		context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-ay")
 	}
 
+	// Activate all VGs on the server
+	context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-ay")
```

Comment:
I think we should consider using just `vgchange` and letting `exec.Cmd` search the `PATH` for where `vgchange` actually lives so we can support environments where this might be a different path.

Reply:
Will implement to search through `PATH`, though to note this will be executed inside the Rook Ceph container image used in the OSD init container.

Reply:
That is not necessary. If `ExecuteCommand` uses `exec.Cmd`, then Go will automatically search `PATH`. https://godoc.org/os/exec#LookPath